frontend/lib/components/NavBar/components/NavItems/components/BrainsDropDown/AddBrainModal.tsx

```diff
@@ -0,0 +1,61 @@
+import Button from "@/lib/components/ui/Button";
+import Field from "@/lib/components/ui/Field";
+import Modal from "@/lib/components/ui/Modal";
+import { useBrainContext } from "@/lib/context/BrainProvider/hooks/useBrainContext";
+import { FC, FormEvent, useState } from "react";
+import { MdAdd } from "react-icons/md";
+
+interface AddBrainModalProps {}
+
+const AddBrainModal: FC<AddBrainModalProps> = ({}) => {
```

Comment:
1/ use Arrow functions with JSX.Element instead of FC (follow other components guidelines for code unicity) (should be caught by eslin, weird)
2/ If no props, no need to define the interface